pkg/gatewayconfigurationserver/server.go

```diff
@@ -166,6 +69,9 @@ func New(c *component.Component, conf *Config) (*Server, error) {
 	v2GCS := gcsv2.New(c, gcsv2.WithTheThingsGatewayConfig(conf.TheThingsGateway))
 	_ = v2GCS
 
+	hooks.RegisterUnaryHook("/ttn.lorawan.v3.GCs", rpclog.NamespaceHook, rpclog.UnaryNamespaceHook("gatewayserver"))
+	hooks.RegisterUnaryHook("/ttn.lorawan.v3.GCs", cluster.HookName, c.ClusterAuthUnaryHook())
```

Comment:
The service is `GatewayConfigurationService` so shouldn't that be registered in the hook?

Reply:
Wasn't sure if you were referring to the namespace name or the `GCs` in the filter. 

Either way I changed the filter to refer to the `GatewayConfigurationService` instead of `GCs` and changed the namespace from `gatewayserver` to `gatewayconfigurationserver`.